rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -215,6 +215,8 @@ data class StringValue(var value: String, var varying: Boolean = false) : Abstra
             else -> super.compareTo(other)
         }
 
+    fun withoutVarying() = if (varying) value.trimEnd() else value
```

Comment:
I think the right name could be `trimEndIfVarying,` but that's just my point of view. 
If the name is too descriptive, we run the risk of needing to change it if the implementation changes. My approach is to declare the visibility as private or internal whenever possible.